### ISSUE TITLE
PackageManagement not installing some packages

### ISSUE THREAD

**Issue by pandanautinspace** [OneGet/oneget#271]
## Hi!
For some reason PackageManagement has simply stopped installing packages for me. It kind of acts like it does, but seems to not show the downloading animation, or the step by step installing process.  Instead, it goes through some kind of process, shows the details of the package, and then simply acts like the package is installed. 

When installing cmake, it got all the way to starting the installer before errorring. Git for windows downloaded but did not install. Inkscape shows nothing but acts like installed. The only applications that I have successfully installed with PackageManagement are python 2.7, Canopy, and NetBeans.

All of those are using the Chocolatey Package Provider, and it turned out that the install script for git for windows was in Chocolatey folder, but not executed...

**Comment by breasor** [comment]
Same thing happening to me. Running install-package -name firefox -debug yields the following: 

VERBOSE: Exception : DefaultDomain//CmdletInvocationException/The term 'Get-WebContent' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

**Comment by breasor** [comment]
also some packages I have downloaded are corrupt (filezilla, winscp.install)

**Comment by brywang-msft** [comment]
@pandanautinspace or @breasor - can you provide OS/PSGet/PackageManagement/PSVersion version info please? In the meantime I'll try to repro.

**Comment by brywang-msft** [comment]
Assuming @breasor 's issue is also from the chocolatey feed + chocolatey provider (since I think Get-WebContent is a chocolatey-core.extension cmdlet), I need to ask some people about chocolatey provider support. Hopefully I can get a good answer where we can transition the provider to official support.

I wasn't able to repro either issue - if either of you can give me more info to help me get a repro, that would be greatly appreciated (otherwise I'm just stumbling around in the dark).

@pandanautinspace - you say "stopped" - does that mean you were able to install chocolatey packages from the chocolatey feed before you upgraded/added/removed... something?

**Comment by breasor** [comment]
OS: Windows 10 Enterprise
PSVersion: 5.1.14393.693

**Comment by pandanautinspace** [comment]
I was incorrect in saying stopped working, it seems that some packages do install and some don't. For example installing git with the chocolatey provider left a git install application in C:/Chocolatey/lib/Git.../tools, which I used to actually install git.

Chocolatey/bin now has node and git in it,two things I was trying to install but ended up installing seperately. I tried installing wget, it failed due to an unrelated error( UnsupportedArchive,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage), cmake now fails to install by an error instead of being silent(I don't know what changed, logfile was empty). Trying to install with -debug leads me to believe the problem with cmake could be because it is a 64bit package on 32bit os.  Despite those, Autohotkey installs fine. 

OS: Microsoft Windows 10 Pro
PSVersion: 5.1.14393.576

**Comment by gaelcolas** [comment]
That sounds related to this: https://github.com/OneGet/oneget/issues/277#issuecomment-302044901

Using the current Chocolatey Provider is unreliable (less and less packages will work with), using the ChocolateyGet is probably a better option.

**Comment by ferventcoder** [comment]
tl;dr - what @gaelcolas mentioned, plus https://github.com/chocolatey/chocolatey-oneget/issues/5#issuecomment-275404099

**Comment by jianyunt** [comment]
Chocolatey provider is a prototype provider created by OneGet team and is on path for deprecation. Please work with Chocolatey owners to get official support.
